psapi.go

```diff
@@ -98,3 +99,33 @@ func EnumProcesses() (pids []uint32, err error) {
 		}
 	}
 }
+
+// PerformanceInformation represents the PERFORMANCE_INFORMATION structure
+type PerformanceInformation struct {
+	CB                uint32
+	CommitTotal       uintptr
+	CommitLimit       uintptr
+	CommitPeak        uintptr
+	PhysicalTotal     uintptr
+	PhysicalAvailable uintptr
+	SystemCache       uintptr
+	KernelTotal       uintptr
+	KernelPaged       uintptr
+	KernelNonpaged    uintptr
+	PageSize          uintptr
+	HandleCount       uint32
+	ProcessCount      uint32
+	ThreadCount       uint32
+}
+
+// GetPerformanceInfo retrieves performance information for the system
+// https://learn.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-getperformanceinfo
+func GetPerformanceInfo() (PerformanceInformation, error) {
+	var pi PerformanceInformation
+	pi.CB = uint32(unsafe.Sizeof(pi))
+
+	if err := _GetPerformanceInfo(&pi, pi.CB); err != nil {
+		return PerformanceInformation{}, fmt.Errorf("GetPerformanceInfo failed: %w", err)
+	}
+	return pi, nil
```

Comment:
I think returning a pointer is much more conventional. Unless there is some reason to return the value, I'd go for a pointer.

```suggestion
func GetPerformanceInfo() (*PerformanceInformation, error) {
	var pi PerformanceInformation
	pi.CB = uint32(unsafe.Sizeof(pi))

	if err := _GetPerformanceInfo(&pi, pi.CB); err != nil {
		return nil, fmt.Errorf("GetPerformanceInfo failed: %w", err)
	}
	return &pi, nil
```